primitives/src/pcs/mod.rs

```diff
@@ -27,12 +28,8 @@ use errors::PCSError;
 /// Note that for our usage, this PCS is not hiding.
 /// TODO(#187): add hiding property.
 pub trait PolynomialCommitmentScheme {
-    /// Prover parameters
-    type ProverParam: Clone;
-    /// Verifier parameters
-    type VerifierParam: Clone + CanonicalSerialize + CanonicalDeserialize;
```

Comment:
There were there because previously `Self::SRS` didn't have `StructuredReferenceString` trait bound. 
And main reason it doesn't have that bound is because in the multilinear kzg implementation, the SRS type is a tuple of MLE_SRS and Uni_SRS, which is also why you see that after adding the trait bound, I need to implement that trait for the tuple. 

This is done in the [first commit](https://github.com/EspressoSystems/jellyfish/pull/231/commits/c268f7099610f30b42e74e13aadc583d8a4991e8) in this PR.